perf/Makefile

```diff
@@ -49,3 +57,39 @@ regen_key_samples:
 
 pkeyread: pkeyread.c keys.h libperf.a
 	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o pkeyread pkeyread.c -lperf -lcrypto
+
+pkeyread-dh-pem: pkeyread
```

Comment:
What about converting this lot into:
```make
pkeyread-%: pkeyread
	$(LN) -s pkeyread $@
```

or if the suggestion about using a variable to hold them, use that on the left of the colon.